annotator-core/src/main/java/edu/ucr/cs/riple/core/util/Utility.java

```diff
@@ -82,6 +83,23 @@ public static void executeCommand(Config config, String command) {
     }
   }
 
+  /**
+   * Clears the content of the file at the given path if the file exists.
```

Comment:
Why don't you just delete the file?

Reply:
Sure, that sounds better. Fixed in 6dd203efc9ca353952a2e8768243772a626ddf8c